demos/simple-gotracing/http_trace/http_trace.go

```diff
@@ -0,0 +1,107 @@
+package main
+
+import "C"
+import (
+	"bytes"
+	"encoding/binary"
+	"flag"
+	"fmt"
+	"os"
+	"os/signal"
+	"unsafe"
+
+	"github.com/fatih/color"
+	"github.com/iovisor/gobpf/bcc"
+)
+
+/*
+#cgo CFLAGS: -I/usr/include/bcc/compat
+#cgo LDFLAGS: -lbcc
+#include <bcc/bcc_common.h>
+#include <bcc/libbpf.h>
+#include <bcc/bcc_syms.h>
+*/
+import "C"
+
+// GoHTTPResponseEvent is the struct written to the perf buffer. It needs to back exactly with definition in the BPF program.
+type GoHTTPResponseEvent struct {
+	StatusCode uint64
+	URILen     uint64
+	URI        [64]byte
+
+	MethodLen uint64
+	Method    [8]byte
+
+	MsgLen uint64
+	Msg    [128]byte
+}
+
+var binaryProg string
+
+func init() {
+	flag.StringVar(&binaryProg, "binary", "", "The binary to probe")
+}
+
+func main() {
```

Comment:
WDYT adding a short description of how to build and run this program.

I suspect this program will live longer than the blog post's attention. An instruction helps anyone who wants to play with it in the future.